pyproject.toml

```diff
@@ -18,3 +18,7 @@ showcontent = true
 directory = "bugfix"
 name = "Bug fixes:"
 showcontent = true
+
+[tool.black]
```

Comment:
I haven't seen this one yet.
Note to self: read https://black.readthedocs.io/en/stable/index.html
Is this what caused the change from single to double quotes though?

Reply:
Apparently: yes. See rationale at https://black.readthedocs.io/en/stable/the_black_code_style.html#strings.
cc @gforcada

Reply:
That's one of the reason I prefer https://github.com/google/yapf over black.

Reply:
I, and we are using that at work, would run black with the -S switch and keep the flake8 pluguins we are using for keeping the single quotes consistency

Reply:
which two lines? :thinking:  :sweat_smile:

Reply:
lines 23 and 24 in this file - `skip-string-normalization = true` means the same as `-S` on command line.

Reply:
:sweat_smile: sorry as you pointed to the black docs I thought they were there.

As for these two lines, I would remove the line length one, it is specially annoying for long imports, and as black takes care of it anyway, let's not bang our heads against it (IMHO)